.github/scripts/install_dependencies.sh

```diff
@@ -31,34 +31,36 @@ sudo apt install -y \
   libxft-dev \
   libxml++2.6-dev \
   libreadline-dev \
-  tcl-dev \
+  tcl8.6-dev \
   libffi-dev \
   perl \
   texinfo \
   time \
   valgrind \
   zip \
-  qt5-default \
+  qtbase5-dev \
   uuid-dev \
   default-jdk \
-  clang-format-7 \
-  g++-7 \
-  gcc-7 \
-  g++-8 \
-  gcc-8 \
+  clang-format-12 \
   g++-9 \
   gcc-9 \
   g++-10 \
   gcc-10 \
   g++-11 \
   gcc-11 \
-  clang-6.0 \
-  clang-7 \
   clang-10 \
```

Comment:
![image](https://user-images.githubusercontent.com/11499826/228431798-92daeefa-c2e8-4055-bf81-39a1d441c6c9.png)

Please try to find the correct package name of clang-10